LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/variations/at-font-face-descriptors-expected.txt

```diff
@@ -71,9 +71,9 @@ PASS font-style(valid): 'oblique' followed by zero degrees: oblique 0deg
 FAIL font-style(valid): 'oblique' followed by default 20deg angle: oblique 20deg assert_equals: Unexpected resulting value. expected "oblique" but got "oblique 20deg"
 PASS font-style(valid): 'oblique' followed by maxumum 90 degree angle: oblique 90deg
 PASS font-style(valid): 'oblique' followed by minimum -90 degree angle: oblique -90deg
-FAIL font-style(valid): 'oblique' followed by calc with out of range value (should be clamped): oblique calc(91deg) assert_equals: Unexpected resulting value. expected "oblique 90deg" but got "oblique calc(91deg)"
-FAIL font-style(valid): 'oblique' followed by calc with out of range value (should be clamped): oblique calc(-91deg) assert_equals: Unexpected resulting value. expected "oblique -90deg" but got "oblique calc(-91deg)"
-FAIL font-style(valid): 'oblique' followed by  angle in radians: oblique 0rad assert_equals: Unexpected resulting value. expected "oblique 0deg" but got "oblique 0rad"
+PASS font-style(valid): 'oblique' followed by calc with out of range value (should be clamped): oblique calc(91deg)
+PASS font-style(valid): 'oblique' followed by calc with out of range value (should be clamped): oblique calc(-91deg)
+FAIL font-style(valid): 'oblique' followed by angle in radians: oblique 0rad assert_equals: Unexpected resulting value. expected "oblique 0deg" but got "oblique 0rad"
```

Comment:
I wonder if this is correct!?

Reply:
@darinadler This seems correct according to the spec, the spec says the canonical unit for `<angle>` is `deg`, which means it should always be used for angles.

https://w3c.github.io/csswg-drafts/css-values/#angles

> All [<angle>](https://w3c.github.io/csswg-drafts/css-values/#angle-value) units are [compatible](https://w3c.github.io/csswg-drafts/css-values/#compatible-units), and [deg](https://w3c.github.io/csswg-drafts/css-values/#deg) is their [canonical unit](https://w3c.github.io/csswg-drafts/css-values/#canonical-unit).

Reply:
OK, I will look into fixing that.

Reply:
Oh I missed that, we probably want to change the test then. There is some discussion around how we should consider `@font-face` descriptors in https://github.com/w3c/csswg-drafts/issues/7964 , since they're not clearly "computed" or "specified", but I lean towards "specified" given the way they're being set/read in this test.